### PR TITLE
Added hyperlinks for address, email, and phone

### PR DESCRIPTION
**Pull request by KT-277**
Have added the hyperlinks of address, email and phone number

### CODE REVIEW

File: home/templates/includes/navigation.html
Comment (s-bonnor) [inline home/templates/includes/navigation.html:1]: This looks like your reverting the recent fixes to the Nav Bar and Dark Mode toggle
could you undo the changes in this file
(possibly just by copying the file from main)

File: home/templates/pages/Contactus.html [hunk 1]
```diff
@@ -19,23 +17,23 @@ <h1>Contact us</h1>
           <p>For further information, please contact us using the details below:</p>
           <div class="icon-text">
             <i class="fas fa-map-marker-alt"></i>
-            <span> 221 Burwood Hwy, Burwood VIC 3125</span>
+            <span><a href="https://g.co/kgs/abekme1" target="_blank">221 Burwood Hwy, Burwood VIC 3125</a></span>
           </div>
```
Comment (s-bonnor) [inline home/templates/pages/Contactus.html:21]: I don't think a short link to a search for the address on google.co.in is the correct thing to put here, could you try and get a maps.google.com link for the address instead

File: home/templates/pages/Contactus.html [hunk 2]
```diff
@@ -19,23 +17,23 @@ <h1>Contact us</h1>
...
           <div class="icon-text">
             <i class="fas fa-phone"></i>
-            <span> 04******</span>
+            <span><a href="tel:+614******">04******</a></span> <!-- Phone number link -->
           </div>
           <div class="icon-text">
             <i class="fas fa-envelope"></i>
-            <span> hardhat@gmail.com</span>
+            <span><a href="mailto:hardhat@gmail.com">hardhat@gmail.com</a></span>
```
Comment (s-bonnor) [inline home/templates/pages/Contactus.html:28]: We previously established that no one at Deakin actually has control over the email address used here,
could you replace it with a placeholder like the phone number

**Review by s-bonnor — CHANGES_REQUESTED** [review]

File: home/templates/pages/Contactus.html (hunk 2 above)
Comment (KT-277) [reply home/templates/pages/Contactus.html:28]: Changed the text from hardhat@gmail.com to hardhat@example.com

File: home/templates/pages/Contactus.html (hunk 1 above)
Comment (KT-277) [reply home/templates/pages/Contactus.html:21]: Changed the link as per the requirements

File: home/templates/includes/navigation.html
Comment (KT-277) [reply home/templates/includes/navigation.html:1]: Changed the navigation.html to the same as it was
